asab/library/providers/git.py

```
@@ -293,7 +303,7 @@ def init_task():
 						callbacks=callbacks
 					)
 				except Exception as err:
```

Ruff (0.14.14)

Do not catch blind exception: `Exception`

(BLE001)